lib/mix/lib/mix/tasks/new.ex

```diff
@@ -147,65 +153,74 @@ defmodule Mix.Tasks.New do
     in the umbrella project root will automatically run
     for each application in the apps/ directory.
     """
-    |> String.trim_trailing
-    |> Mix.shell.info()
+    |> String.trim_trailing()
+    |> Mix.shell().info()
   end
 
   defp check_application_name!(name, inferred?) do
     unless name =~ Regex.recompile!(~r/^[a-z][a-z0-9_]*$/) do
-      Mix.raise "Application name must start with a letter and have only lowercase " <>
-                "letters, numbers and underscore, got: #{inspect name}" <>
-                (if inferred? do
-                  ". The application name is inferred from the path, if you'd like to " <>
-                  "explicitly name the application then use the \"--app APP\" option"
-                else
-                  ""
-                end)
+      Mix.raise(
+        "Application name must start with a letter and have only lowercase " <>
+          "letters, numbers and underscore, got: #{inspect(name)}" <>
+          if inferred? do
+            ". The application name is inferred from the path, if you'd like to " <>
+              "explicitly name the application then use the \"--app APP\" option"
+          else
+            ""
+          end
+      )
     end
   end
 
   defp check_mod_name_validity!(name) do
     unless name =~ Regex.recompile!(~r/^[A-Z]\w*(\.[A-Z]\w*)*$/) do
-      Mix.raise "Module name must be a valid Elixir alias (for example: Foo.Bar), got: #{inspect name}"
+      Mix.raise(
+        "Module name must be a valid Elixir alias (for example: Foo.Bar), got: #{inspect(name)}"
+      )
     end
   end
 
   defp check_mod_name_availability!(name) do
     name = Module.concat(Elixir, name)
+
     if Code.ensure_loaded?(name) do
-      Mix.raise "Module name #{inspect name} is already taken, please choose another name"
+      Mix.raise("Module name #{inspect(name)} is already taken, please choose another name")
     end
   end
 
   defp check_directory_existence!(path) do
-    if File.dir?(path) and not Mix.shell.yes?("The directory #{inspect(path)} already exists. Are you sure you want to continue?") do
-      Mix.raise "Please select another directory for installation"
+    if File.dir?(path) and
+         not Mix.shell().yes?(
+           "The directory #{inspect(path)} already exists. Are you sure you want to continue?"
```

Comment:
Can you please move the message to a variable that is defined before the `if`? Thanks.

Reply:
Done. I hope `msg` is a valid variable name here, since it's a rather short function.